mule-migration-tool-e2e-tests/src/test/resources/e2e/gaps/unknown_namespace/output/report/report.json

```diff
@@ -38,22 +38,24 @@
     {
       "level": "ERROR",
       "key": "components.unsupported",
-      "component": "unknown:source",
-      "lineNumber": 5,
-      "columnNumber": 25,
-      "message": "The migration of \u0027unknown\u0027 is not supported.",
+      "component": "flow",
```

Comment:
should report `unknown:source` as the error component with its line and col numbers